tests/node_account/test_get.py

```diff
@@ -0,0 +1,201 @@
+"""
+Provide tests for command line interface's node account information by its address commands.
+"""
+import json
+
+import pytest
+from click.testing import CliRunner
+
+from cli.constants import (
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    NODE_IP_ADDRESS_FOR_TESTING,
+    PASSED_EXIT_FROM_COMMAND_CODE,
+)
+from cli.entrypoint import cli
+from cli.utils import dict_to_pretty_json
+
+NODE_ACCOUNT_ADDRESS_PRESENTED_ON_THE_TEST_NODE = \
+    '1168290a2cbbce30382d9420fd5f8b0ec75e953e5c695365b1c22862dce713fa1e48ca'
+
+
+def test_get_information_with_address():
+    """
+    Case: get information about the node account by its address.
+    Expect: information about the node account is returned.
+    """
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'node-account',
+        'get',
+        '--address',
+        NODE_ACCOUNT_ADDRESS_PRESENTED_ON_THE_TEST_NODE,
+        '--node-url',
+        'node-1-testnet.remme.io',
```

Comment:
Move to the constant, please. When finally we get the one node for tests, we will refactor it easier with constants.

Reply:
@dmytrostriletskyi, done.